test/functional/rpc_blockchain.py

```diff
@@ -241,6 +245,15 @@ def _test_gettxoutsetinfo(self):
         del res['disk_size'], res3['disk_size']
         assert_equal(res, res3)
 
+        self.log.info("Test that MuHash implementation in Python returns the same result as C++")
```

Comment:
This should be a unit test in muhash.py. See #18576 for details on the python unit tests.

Reply:
done